tests/dummy/config/environment.js

```diff
@@ -16,9 +16,19 @@ module.exports = function(environment) {
     APP: {
       // Here you can pass flags/options to your application instance
       // when it is created
+    },
+
+    twitch: {
+      redirectURI: 'http://localhost:4233/twitch-auth',
+      clientId: 'e18ok8watl2s2y3t86yhyyibm3ew3k2',
```

Comment:
This might be a good opportunity to use https://github.com/fivetanley/ember-cli-dotenv

That way we can each keep our API keys secret without checking them into the repo.